omnigibson/robots/manipulation_robot.py

```diff
@@ -757,15 +779,15 @@ def _calculate_in_hand_object_rigid(self, arm="default"):
 
         # Make sure at least two fingers are in contact with this object
         robot_contacts = robot_contact_links[ag_prim_path]
-        touching_at_least_two_fingers = len({link.prim_path for link in self.finger_links[arm]}.intersection(robot_contacts)) >= 2
+        touching_at_least_two_fingers = True if self.grasping_mode == "sticky" else len({link.prim_path for link in self.finger_links[arm]}.intersection(robot_contacts)) >= 2
```

Comment:
I think this is actually incorrect. Based on the previous documentation:
```
If "assisted", will magnetize any object touching and within the gripper's fingers.
If "sticky", will magnetize any object touching the gripper's fingers.
```

It seems that "sticky" requires `touching_at_least_two_fingers`. 
Also, it seems that "assisted" has not been implemented yet. 

Should we revert back?

Reply:
I have fixed the documentation.